src/common/src/mlib/config.h

```diff
@@ -147,8 +173,8 @@
  * @brief Expand to a call expression `Prefix##_argc_N(...)`, where `N` is the
  * number of macro arguments.
  */
-#define MLIB_ARGC_PICK(Prefix, ...) \
-   MLIB_JUST (MLIB_PASTE_3 (Prefix, _argc_, MLIB_ARG_COUNT (__VA_ARGS__)) (__VA_ARGS__))
+#define MLIB_ARGC_PICK(Prefix, ...) MLIB_ARGC_PASTE (Prefix, __VA_ARGS__) (__VA_ARGS__)
+#define MLIB_ARGC_PASTE(Prefix, ...) MLIB_PASTE_3 (Prefix, _argc_, MLIB_ARG_COUNT (__VA_ARGS__))
```

Comment:
It looks like there may be something about this change (or related) which MSVC doesn't like:

```
warning C4003: not enough arguments for function-like macro invocation '_mlib_foreach_urange_argc_3'
error C2059: syntax error: 'constant'
```

Reply:
Uggh. That's the old broken preprocessor trying to pass the whole `__VA_ARGS__` as a single macro argument again. It's a trivial fix.